src/deps.js

```diff
@@ -1,3 +0,0 @@
-// This file was autogenerated by third_party/closure-library/closure/bin/build/depswriter.py.
-// Please do not edit.
-goog.addDependency('src/helper/helper.js', ['helper'], ['plain']);
```

Comment:
Is this deleting the file, or just the content? I think deleting the file is a good idea.

Reply:
I think I was deleting it, but I was just talking with KJ yesterday and she may need it because she's trying to run the tests with closure compiler from karma with transpilation down to ES5 for IE, and it may or may not be needed for her stuff. So I put all of it back in for now